docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,530 @@
+lido = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84
+legacyOracle = 0x442af784A788A5bd6F42A01Ebe9F287a871243fb
+```
+
+## ValidatorsExitBusOracle
+And its corresponding `HashConsensus`.
+
+```python
+# Thrice per day
+# So, the ValidatorsExitBusOracle expected report time would be 
+# ~4:00 UTC, ~12:00 UTC, ~20:00 UTC
+epochsPerFrame = 75
+
+# Number of slots dedicated for delay during oracles rotation including finalization time
+# https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/oracle/HashConsensus.sol#L370-L398
+# NB: min value is 64 as two epochs are required for finalization
+fastLaneLengthSlots = 100
+
+# Technical value indicating consensus version of the contract. Can basically be any because there were no consensus versions before https://github.com/lidofinance/lido-dao/blob/e35435ad9b9473cb0f9b7b0e1e17f6bf9b96e000/contracts/0.8.9/oracle/BaseOracle.sol#L121-L126
+consensusVersion = 1
+
+lidoLocator = 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
+```
+
+## AccountingOracle and ValidatorsExitBusOracle
+
+```python
+# See https://ethereum.org/en/developers/docs/blocks/
+secondsPerSlot = 12
+
+# See https://blog.ethereum.org/2020/11/27/eth2-quick-update-no-21
+# Also its as it is in the current oracle https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb#readProxyContract#F30
+genesisTime = 1606824023
+```
+
+## WithdrawalQueueERC712
+
+```python
+_wstETH = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0
+_name = "Lido: stETH Withdrawal NFT"
+_symbol = "unstETH"
+
+# Lido-maintained NFT-generator server 
+# see https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/WithdrawalQueueERC721.sol#L126-L129
+_tokenBaseURI = "https://wq-api.lido.fi/v1/nft"
+```
+
+## WithdrawalsVault
+
+```python
+_lido = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84 
+_treasury = 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c  # Agent
+```
+
+## EIP712StETH
+
+```python
+_stETH = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84
+```
+
+## StakingRouter
+
+```python
+# See https://ethereum.org/en/staking/deposit-contract/
+_depositContract = 0x00000000219ab540356cBB839Cbe05303d7705Fa
+```
+
+## GateSeal
+
+```python
+# 2 x governance response time - vote duration is 3 days
+seal_duration = 518400  # 6 days as seconds
+expiry_timestamp = 1714521600  # 2024-05-01 00:00 UTC
+```
+
+## Roles setup
+
+Contracts ACL denotion:
+- mark "*Aragon app*" means the contract is Aragon app which uses [Aragon ACL model](https://hack.aragon.org/developers/tools/aragonos/reference-documentation)
+- mark "*OZ*" means the contract uses [OpenZeppelin ACL model](https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/utils/access/AccessControlEnumerable.sol)
+- mark "*Proxy*" means the contract is deployed behind a proxy so has an additional [ACL model related to the proxy](https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/proxy/OssifiableProxy.sol)
```

Comment:
Let's add the `Plain owner` mark to the list (DSM case)
and 'No access control' (WithdrawalVault case)